prow/cmd/validation-webhook/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/test-infra/experiment/clustersecretbackup/secretmanager"
+	"k8s.io/test-infra/prow/logrusutil"
+)
+
+const (
+	secretID  = "ca-certification"
+	secretName = "ca-secrets"
+	caCert = "ca-cert"
+	caPrivKey = "ca-priv-key"
+	caBundle = "ca-bundle"
+)
+
+func main() {
+	logrusutil.ComponentInit()
+	logrus.SetLevel(logrus.DebugLevel)
+	// GCP project in which to store secrets in Secret Manager
+	projectId := ""
+	secretsMap := make(map[string]string)
+	client, err := secretmanager.NewClient(projectId, false)
+	ctx := context.Background()
+	if err != nil {
+		logrus.WithError(err).Fatal("Unable to create secret manager client")
+	}
+
+	data, err := client.GetSecretValue(ctx, secretID, secretName)
+	if err != nil {
+		logrus.WithError(err).Fatal("Unable to get secret data")
+	}
+
+	err = json.Unmarshal(data, &secretsMap)
+	if err != nil {
+		logrus.WithError(err).Fatal("Unable to unmarshal secrets data")
+	}
+
+	cert := secretsMap[caCert]
+	privKey := secretsMap[caPrivKey]
+
+	if !isCertValid(cert) {
+		createGCPSecret(client, ctx)
+		runServer(cert, privKey)
+	} else {
+		runServer(cert, privKey)
+	}
+
+}
+
+func runServer(cert string, key string) {
+	http.HandleFunc("/validate", serveValidate)
+	logrus.Print("Listening on port 8008...")
```

Comment:
I'm not aware of the usage of `logrus.Print`, more commonly used are `.Info`, `.Warn`, `.Error` etc.